hudi-common/src/main/java/org/apache/hudi/common/bootstrap/index/HFileBootstrapIndex.java

```diff
@@ -238,7 +239,7 @@ private static HFileReader createReader(String hFilePath, FileSystem fileSystem)
       LOG.info("Opening HFile for reading :" + hFilePath);
       Path path = new Path(hFilePath);
       long fileSize = fileSystem.getFileStatus(path).getLen();
-      FSDataInputStream stream = fileSystem.open(path);
+      SeekableDataInputStream stream = new HadoopSeekableDataInputStream(fileSystem.open(path));
```

Comment:
Is this going to be HadoopSeekableDataInputStream going forward? Or is hadoop going to be fully removed from here at some point?

Reply:
This will be replaced by the new storage API call which returns `SeekableDataInputStream` directly.  Hadoop is going to be fully removed here in the future.